test/test-applications/integrations/TestApplication.GraphQL.NativeSupport/StarWars/Types/HumanType.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="HumanType.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
```

Comment:
Just FYI, this file, and other of the GraphQL sample code copied into this PR, are direct copies from the GraphQL.NET repo.  They are copyrighted by the GraphQL team and require that "[the] copyright notice and [the] permission notice shall be included in all copies or sustantial portions of the Software.".  You may sublicense the code under the Apache License, but the GraphQL license must be included as well if the GraphQL source is included here.  (Note that the GraphQL license copied herein would not apply to modifications or any other code by the OpenTelemetry team, but simply relates to the code sourced from the GraphQL team.)

Since you are explicitly listing the copyright in each file, I suggest you include the other license in these specific files as well, stating "Portions of these files copyrighted as follows:" with the GraphQL.NET copyright and permission notice listed.  Alternatively, you could replace the entire copyright notice with the GraphQL.NET notice.

Reply:
Excellent notice! Thank you.

@Shane32 I noticed this was previously a separate package, publicly available in Nuget. Are there any ways for the latest, not to bring this source but instead reference?

Reply:
https://github.com/cncf/foundation/blob/main/copyright-notices.md#what-about-third-party-code

According to this seems I can actually remove OTel copyright header. 

@Shane32 Is there a license file somewhere? - I could add it into the `Starwars` folder and remove OTel copyrights.

Reply:
See:
- https://github.com/graphql-dotnet/graphql-dotnet/blob/master/LICENSE.md

Reply:
Ah that was just one folder up from where I was looking, thanks!